### PR TITLE
Proper fix for icon scaling preferences

### LINKED ISSUE #27281: Icon scaling mode preference never saves
Even if you change to something other than nearest neighbor, it will reset.

@MrStonedOne just in case you forgot!

### PR DESCRIPTION
**Pull request by MrsTonedOne**
fixes #27281

:cl: MrStonedOne and Lummox JR
fix: Fixed icon scaling and size preferences not loading (Technically size preferences were loading because byond saves that locally, but that's not reliable because ss13 shares a hub)
/:cl:

### CODE REVIEW

File: interface/menu.dm [hunk 1]
```diff
@@ -68,10 +68,11 @@ GLOBAL_LIST_EMPTY(menulist)
 	var/atom/verb/verbpath = Get_checked(C)
 	if (!verbpath || !(verbpath in typesof("[type]/verb")))
 		return
+	
 	if (copytext(verbpath.name,1,2) == "@")
-		winset(C, null, "command = [copytext(verbpath.name,2)]")
+		winset(C, null, list2params(list("command" = copytext(verbpath.name,2))
```
Comment (lzimann) [inline interface/menu.dm]: missing a `)` in both

Comment (MrStonedOne) [inline interface/menu.dm]: WRONG

Comment (MrStonedOne) [inline interface/menu.dm]: FAKE NEWS

Comment (MrStonedOne) [inline interface/menu.dm]: Clearly I'm missing two `)`s

**Pull request merged**